docs/060-breaking-changes.rst

```diff
@@ -42,7 +42,8 @@ For most of the topics the compiler will provide suggestions.
   storage arrays.
 
 * The new keyword ``abstract`` can be used to mark contracts as abstract. It has to be used
-  if a contract does not implement all its functions.
+  if a contract does not implement all its functions. Abstract contracts cannot be initialised and solidity outputs no
```

Comment:
solidity -> Solidity or rather "the compiler outputs no bytecode for them".

Reply:
@axic do you want me to open another PR for this?

Reply:
It has already been fixed, thanks!